Description: Drop support for getting gitlab authentication from authentication.conf.
Origin: commit, revision id: jelmer@jelmer.uk-20190212233825-32367tbws1dr04yn
Author: Jelmer VernooÄ³ <jelmer@jelmer.uk>
Last-Update: 2019-02-12
Applied-Upstream: no
X-Bzr-Revision-Id: jelmer@jelmer.uk-20190212233825-32367tbws1dr04yn

=== modified file 'breezy/plugins/propose/gitlabs.py'
--- old/breezy/plugins/propose/gitlabs.py	2019-02-03 02:23:04 +0000
+++ new/breezy/plugins/propose/gitlabs.py	2019-02-12 23:38:25 +0000
@@ -24,7 +24,6 @@
     errors,
     urlutils,
     )
-from ...config import AuthenticationConfig
 from ...git.urls import git_url_to_bzr_url
 from ...sixish import PY3
 
@@ -100,23 +99,15 @@
 
 def connect_gitlab(host):
     from gitlab import Gitlab, GitlabGetError
-    auth = AuthenticationConfig()
-
     url = 'https://%s' % host
-    credentials = auth.get_credentials('https', host)
-    if credentials is None:
-        for name, section in iter_tokens():
-            if section.get('url') == url:
-                credentials = section
-                break
-        else:
-            try:
-                return Gitlab(url)
-            except GitlabGetError:
-                raise GitLabLoginMissing()
+    for name, section in iter_tokens():
+        if section.get('url') == url:
+            return Gitlab(**section)
     else:
-        credentials['url'] = url
-    return Gitlab(**credentials)
+        try:
+            return Gitlab(url)
+        except GitlabGetError:
+            raise GitLabLoginMissing()
 
 
 def parse_gitlab_url(branch):

